Makefile

```diff
@@ -1390,9 +1390,13 @@ flash: FORCE
 endif
 
 ifneq ($(DEBUG_SCRIPT),)
+ifeq ($(USE_ZEPHYR_FLASH_DEBUG_SHELL),)
```

Comment:
is there a reason not to just use your script always?

Reply:
With this series, the script is being used by default now for everything (flash, debug, debugserver).

I'm leaving this in as an escape hatch in case there are problems after the merge, the same way as was done when I re-did the flashing scripts in https://github.com/zephyrproject-rtos/zephyr/pull/4304.

My plan is to remove the shell scripts once these get merged and a few days of testing, but I an remove them at the end of this series as well if that's better.

Reply:
it's straightforward enough to revert a PR if it causes problems.
problem with checks like this, in my experience they don't always get removed and become cruft.

Reply:
I went ahead and deleted all the shell scripts in v2.